Okafor.Irene-okaforirenen/scripts/index.js

```diff
@@ -0,0 +1,216 @@
+    loader.style.display="block";
+    GetCatPage(category); 
+    pageContent.style.display = "block";  
+}
+
+function Hidepage() 
+{
+    
+    carousel.style.display = "block"
+    trending.style.display = "block";
+    pageContent.style.display = "none";
+}
+
+
+// loaderSection
+
+const loader = document.getElementById("loader");
+
+function GetPage() 
+{
+    let url = 'https://newsapi.org/v2/top-headlines?' +
+              'country=us&' +
+              'apiKey=95b59d83c586471e8438bc6e0eb91bba';
+  
+    let req = new Request(url);
+   
+    fetch(req)
+        .then(resp => resp.json())
+        .then(data => 
+            {
+            let doc = document.createDocumentFragment();
+                [...data.articles].forEach( article =>
+                    {
+                    let panel = document.createElement("div");                   
+                    let imagediv = document.createElement("div");
+                    let image = document.createElement("img");
+                    let desc = document.createElement("div");
+                    let link = document.createElement("a");
+                  
+                    panel.classList.add("panel"); 
+                    imagediv.classList.add("panel-img"); 
+                    desc.classList.add("panel-desc");
+                    link.classList.add("read-more");
+
+                    image.src = article["urlToImage"];
+                    
+                    imagediv.appendChild(image);
+                    desc.innerHTML = article["title"];
+                    link.href = article["url"];
+                    link.target="_blank";
+                    link.innerHTML = "Read More...";
+
+
+                    panel.appendChild(imagediv);
+                    panel.appendChild(desc)
+                    panel.appendChild(link);  
+                    doc.appendChild(panel)   
+
+                    }
+                    
+                    )
+          
+            trending.append(doc);
+            loader.style.display = "none";
+
+        })
+        .catch(err => 
+            {
+                console.log(err);
+            trending.innerHTML =`<h1 class="error">Unable to retrieve news articles. Try refreshing the page<h1>`
+            loader.style.display = "none";          
+            })    
+        
+    
+}
+
+
+
+
+
+
+function GetCatPage(category) 
+{
+   
+
+    let url = `https://newsapi.org/v2/top-headlines?country=us`+
+               `&category=${category}`+
+               `&apiKey=95b59d83c586471e8438bc6e0eb91bba`;    
+    pageContent.innerHTML = null;
+    const homebtn = document.createElement('span');
+    homebtn.innerHTML = " Home &#8629;";
+    homebtn.setAttribute("id", "home-btn");
+    pageContent.appendChild(homebtn);
+    homebtn.addEventListener("click", Hidepage, false);
+
+
+    // https://newsapi.org/v2/top-headlines?country=us&category=business&apiKey=95b59d83c586471e8438bc6e0eb91bba
+    let req = new Request(url);
```

Comment:
You do not need to create a new Request when you're not adding network options, you can just the url with fetch. I'm not saying you should change it though. Just pointing it out